_unittest/test_22_Circuit_DynamicLink.py

```diff
@@ -210,31 +210,34 @@ def test_09_setup(self):
         assert LNA_setup.update()
 
     @pytest.mark.skipif(is_linux, reason="Skipped because Desktop is crashing")
-    def test_10_q3d_link(self):
+    def test_10_q2d_link(self):
         self.aedtapp.insert_design("test_link")
         q2d = Q2d(self.q3d, specified_version=desktop_version)
-        proj_path = self.q3d
         proj_name = q2d.project_name
         c1 = self.aedtapp.modeler.schematic.add_subcircuit_dynamic_link(q2d, extrusion_length=25)
         assert c1
         assert len(c1.pins) == 6
         assert c1.parameters["Length"] == "25mm"
         assert c1.parameters["r1"] == "0.3mm"
-        if proj_name in self.aedtapp.project_list:
-            proj_path = proj_name
-        q3d = Q3d(proj_path, specified_version=desktop_version)
+
+    @pytest.mark.skipif(is_linux, reason="Skipped because Desktop is crashing")
+    def test_10_q3d_link(self):
+        q3d = Q3d(self.q3d, specified_version=desktop_version)
 
         q3d_comp = self.aedtapp.modeler.schematic.add_subcircuit_dynamic_link(
             q3d, solution_name="Setup1 : LastAdaptive"
         )
         assert q3d_comp
         assert len(q3d_comp.pins) == 4
-        hfss = Hfss(proj_path, specified_version=desktop_version)
+
+    @pytest.mark.skipif(is_linux, reason="Skipped because Desktop is crashing")
```

Comment:
@maxcapodi78 Is it crashing because of a bug in PyAEDT or because of a limitation in AEDT?